include/xrpl/basics/base_uint.h

```diff
@@ -623,6 +623,13 @@ to_string(base_uint<Bits, Tag> const& a)
     return strHex(a.cbegin(), a.cend());
 }
 
+template <std::size_t Bits, class Tag>
+inline std::string
+to_short_string(base_uint<Bits, Tag> const& a)
```

Comment:
[nit]: Adding checks for the to_short_string in the base_unit_test next to existing to_string cases would be good.

Reply:
I wouldn't call that a nit. Missing test coverage is pretty significant. Thanks for catching it. Fixed.